pyanaconda/bootloader.py

```diff
@@ -1862,7 +1862,10 @@ class Aarch64EFIGRUB(EFIGRUB):
     _efi_binary = "\\shimaa64.efi"
 
 class MacEFIGRUB(EFIGRUB):
-    _packages64 = [ "grub2-tools-efi", "mactel-boot" ]
+    def __init__(self):
+        super(MacEFIGRUB, self).__init__()
+        self._packages64 = super(MacEFIGRUB, self)._packages64 + ["grub2-tools-efi", "mactel-boot"]
```

Comment:
By doing this you are replacing that list not adding to it.
By assigning something to `self._packages64` variable you are creating `self._packages64` instance (not class) variable so the original array is not extended at all. You really don't want to do this.

Please use the same solution as it was before. Just manually copy & paste old values from the array of parent and extend it by typing new values or you can do this:

``self._packages64 = EFIGRUB._packages64 + ["grub2-tools-efi", "mactel-boot"]``

Sorry it is that way, I think we should change this whole class and replace these class variables by instance variables. I don't think there is a reason for having them as class variables.